VSCodeHelper/VSCodeInstances.cs

```diff
@@ -77,17 +77,24 @@ public static void LoadVSCodeInstances()
             {
                 if (!Directory.Exists(path))
                     continue;
-                
-                var files = Directory.EnumerateFiles(path).Where(x =>
+
+                var newPath = path;
+                if (!Path.GetFileName(path).Equals("bin", StringComparison.OrdinalIgnoreCase))
```

Comment:
Could you explain the changes?

Reply:
Basically, when you install Code using the normal installer, the path will not lead to the root directory of Code, but to the bin subfolder. When you do a portable instance, you will add the root directory to the path, not the bin subfolder. That's why I added a test for all paths not ending in the bin subfolder to simulate the path added by the installer.
So this code will add a bin subfolder to all paths that are not terminated by it, and check if such a path is valid. If so, it will proceed as with the normal Code path added by the installer. This should fix #1 that was closed without fix.

Reply:
Is there a specific reason that we don't want to add the bin subfolder to the path (I think that's mostly how people uses PATH env?).

Reply:
This is mainly because people are used to adding a root directory for any software with a portable instance, if they want to use it from command line. I am convinced that if they knew they had to add a bin subfolder in Code case, they would do it.

I believe this is why one user in #1 told that PowerToysRun works with portable instance, while this plugin is not.